src/Akeneo/Connectivity/Connection/back/Infrastructure/Persistence/Elasticsearch/Query/SearchEventSubscriptionDebugLogsQuery.php

```diff
@@ -221,9 +263,53 @@ private function findSameLastNoticeAndInfoIds(string $connectionCode, string $fi
         return [
             'first_id' => $firstId,
             'first_search_after' => $firstSearchAfter,
-            'ids' => array_merge([$firstId], array_map(function ($hit) {
-                return $hit['_id'];
-            }, $result['hits']['hits'])),
+            'ids' => array_merge(
+                [$firstId],
+                array_map(
+                    function ($hit) {
+                        return $hit['_id'];
+                    },
+                    $result['hits']['hits']
+                )
+            ),
         ];
     }
+
+    private function resolveFilters(array $filters): array
+    {
+        $resolver = new OptionsResolver();
+
+        $resolver->setDefault('levels', null);
```

Comment:
```suggestion
        $resolver->setDefault('levels', null);
```
Could the default be ['info', 'notice', 'warning', 'error'] ?